examples/angular/src/pages/ui/components/authenticator/sign-up/sign-up.component.html

```diff
@@ -1 +1,10 @@
-<amplify-authenticator></amplify-authenticator>
+<amplify-authenticator>
+  <ng-template
+    amplifyOverride="authenticated"
+    let-user="user"
+    let-signOut="signOut"
+  >
+    <h2>Welcome, {{ user.username }}!</h2>
+    <button (click)="signOut($event)">Sign Out</button>
+  </ng-template>
+</amplify-authenticator>
```

Comment:
This is how you can do sign-out within template. Note that we don't need to import any services to do so! Service can be imported for more headless or advanced usage like `state.matches` (or whichever facade we will provide).

Reply:
Does `$event` need to be passed, or is this best-practices because this is an `onClick` HTML event?

Reply:
Oops, nope! Unless you want to do `$event.preventDefault()`, which is not really necessary here. I'll remove it for simplicity.